documentation/intro_forsc.md

```diff
@@ -36,10 +36,13 @@ See {ref}`server-and-env-options` for more information.
 
 ### Storage Resources
 
-The Fornax Science Console offers two types of storage:
+The Fornax Science Console currently offers 3 types of storage:
 
--   **Persistent home directory:** Each user has a 10 GB home directory on the file system for storing important files that need to be saved long term.
--   **Extended temporary storage:** For project work lasting several months, users can request up to 5 TB of object storage on {term}`S3`, intended for data that doesn’t require fast file system access and won’t be stored permanently.
+-   **Persistent home directory:** Each user has a 20 GB private home directory on the file system for storing important files that need to be saved long term. Users can request a size increase incremently up to 100 GB.
+-   **Shared and temporary storage** While a detailed sharing solution is being implemented (planned for early 2026), sharing is currently available via the `~/shared-storage` folder (also accessed from `/shared-storage`). This is a shared efs drive that is writable by all users. To use, create a folder with your name that stores your files. This folder can be used for both sharing and also for short term (~weeks) storage of large files that you want to persist between sessions, but do not fit in the home directory. Note, however, that the content of this folder is visible and writable by all users.
```

Comment:
Should people ask for permission before putting a large amount of data in this directory? (If not, no change needed here.)